plugins/abnormal_security/help.md

```diff
@@ -85,7 +85,7 @@ provided, it will return up to 100 latest results
 
 |Name|Type|Default|Required|Description|Enum|Example|
 | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
-|filter_key|enum - lastModifiedTime - createdTime - customerVisableTime|lastModifiedTime|False|This input enables you to select what timestamp to filter on, default is lastModifiedTime|None|lastModifiedTime|
+|filter_key|string|lastModifiedTime|False|This input enables you to select what timestamp to filter on, default is lastModifiedTime|['lastModifiedTime', 'createdTime', 'customerVisableTime', '']|lastModifiedTime|
```

Comment:
Should this remain as an enum?

Reply:
This needs to be of type string as thats the type of the values in the enum comes from the enum col